docs/src/inverse_problems/behaviour_optimisation.md

```diff
@@ -1,4 +1,4 @@
-# [Optimization for non-data fitting purposes](@id behaviour_optimisation)
+# [Optimization for Non-data Fitting Purposes](@id behaviour_optimisation)
```

Comment:
I realised that almost all pages have capitalisation throughout the titles, but there were a couple where I hadn't done this. Don't really know, but figured I'd change the ones I hadn't done this to be consistent.